components/manager/scripts/validate.py

```diff
@@ -0,0 +1,17 @@
+import os
```

Comment:
Though the functionality is rather the same, perhaps is should be implemented in the cli code rather than in a blueprint. Since the bootstrap process itself uses the rest client regardless of the client usage.